acceptance-tests/bbl/print_env_test.go

```diff
@@ -29,7 +29,7 @@ var _ = Describe("plan", func() {
 
 		stateDir = configuration.StateFileDir
 
-		configuration.BBLStateBucket = "bbl-acceptance-test-states"
+		configuration.BBLStateBucket = "bbl-acceptance-test-states-new"
```

Comment:
Maybe something more descriptive like `bbl-acceptance-test-states-cff` could be used here?